CONTRIBUTING.md

```diff
@@ -132,6 +132,14 @@ Also, please bear the following coding guidelines in mind:
   expansions will be unexpectedly performed, which becomes a vulnerability.  In
   the latter case, checks by shellcheck and shfmt will not be performed inside
   `'...'`.  Also, `_comp_compgen_split` is `IFS`-safe.
+  
+  Avoid using `_comp_compgen -- -G "pattern"` to generate completions.  The
+  result is not filtered by the current word `cur` due to the Bash design of
+  `compgen`.  Also, this cannot be used to generate filenames with a specified
+  extension because the `-G` specification only generates the matching
+  filepaths in the current directory.  It does not look into subdirectories
+  even when `$cur` implies the completion in a subdirectory.  One can instead
+  use `_comp_compgen -- -f -X '!pattern'`.
```

Comment:
This hunk could be in a commit of its own -- now it's "buried" in a `fix(mdtool)` one without a note. No big deal, but thought I'd mention.

Reply:
Thanks! I separated the commit in the push [`df59a49..e98f1a7`](https://github.com/scop/bash-completion/compare/df59a49d00d8e953e0317720444b1fc3742acc36..e98f1a704e8ca17f34979f1f32b79d48092ecd5f).